src/metrics.rs

```diff
@@ -867,6 +915,9 @@ pub struct GooseMetrics {
     /// This value may be smaller than what was configured at start time if the test
     /// didn't run long enough for all configured users to start.
     pub users: usize,
+    /// Number of users at the end of each second of the test. Each element of the vector
+    /// represents one second.
+    pub users_per_second: Vec<u32>,
```

Comment:
Done. Well... the reason was that some of the functions expected `u32` and `Vec<u32>` and I was trying to avoid generic types. Now you forced me to bite into that apple :)